package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nuclia",
-  "version": "1.2.4",
+  "version": "2.0.0",
```

Comment:
Good to update the version, but you forgot to update the CHANGELOG

Reply:
there is no CHANGELOG in the root (that's not the SDK's package here, but the main one)